brabbuss/moodytunes
# Form moods

#### What does  this PR do?
- Adds additional moods to the Form
- Centers the Form submit button
- Matches styling on Form to styling on Favorites
- Makes the "Results" header dynamic so that the genre name is displayed
- Changes label for "content" to "chill"

#### Where should the reviewer start?
- Form
- App
- Favorite
- ResultsView

#### How should this be manually tested?
- Pull down branch and run through the app from Form to FavoritesView
- Choose "Amorous" or "Excited" to ensure that new buttons are working

#### What are the relevant tickets?
Closes #39, Closes #41
